config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users 
```

Comment:
There's an extra space at the end of this line.  Are you using Sublime Text?  Is it version 3?  If so, you should go into your settings (Cmd-comma on Mac, Preferences menu > Settings on Windows) and add the following two lines to the right-hand side (user settings) config:

```
"trim_trailing_white_space_on_save": true,
"ensure_newline_at_eof_on_save": true,
```

So it'll look something like:

```json
{
	"color_scheme": "Packages/Theme - Primer/primer.light.tmTheme",
	"ignored_packages":
	[
		"Vintage"
	],
	"theme": "Primer.sublime-theme",
	"trim_trailing_white_space_on_save": true,
	"ensure_newline_at_eof_on_save": true
}
```

That way whenever you save a file in your editor, it will automatically remove any stray trailing spaces like this.

Let me know if you're using Sublime Text v2 as I think the setting names might be slightly different.